introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs

```diff
@@ -158,9 +158,11 @@ pub(crate) fn calculate_scalar_field(
     let (default_value, dbgenerated_string) = calculate_default(table, &column, &arity);
 
     let (is_commented_out, documentation) = match (&field_type, dbgenerated_string) {
-        (FieldType::Unsupported(_), None) => (true, Some("This type is currently not supported.".to_string())),
+        // (FieldType::Unsupported(_), None) => (true, Some("This type is currently not supported.".to_string())),
+        (FieldType::Unsupported(_), None) => (false, Some("This type is currently not supported.".to_string())),
         (FieldType::Unsupported(_), Some(default)) => (
-            true,
+            // true,
+            false,
```

Comment:
question from review call: does this need to be feature flagged as well? (maybe already feature flagged higher up)

Reply:
the type is checked again higher up and there is a feature flag there